src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceArray = sourceString.split(';');
+  const readyArr = {};
+
+  for (const property of sourceArray) {
+    const propertyArray = property.trim().split(':');
+    const [keyName, keyValue] = propertyArray;
 
+    if (keyValue !== undefined) {
+      readyArr[keyName.trim()] = keyValue.trim();
+    }
+  }
```

Comment:
Instead of using a for-loop, consider using array iteration methods like 'map' and 'reduce' to transform the array of style strings into an object. This will make the code more declarative and align with modern JavaScript practices.